dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -500,6 +570,9 @@ public void addTasksTo(List<? super Task<?>> out, @Nonnull ConnectorArguments ar
       out.add(new DelegationTokensTask());
       out.add(new FunctionsJsonlTask());
       out.add(new ResourcePlansJsonlTask());
+      out.add(new TablesRawJsonlTask(databasePredicate));
```

Comment:
Is there any reason for not including both `TablesJsonTask` and `TablesRawJsonlTask`? My concern is that ZIP extracted with `MIGRATION_METADATA` property won't include `tables.jsonl` file that is required for Hive assessment. User will have to run Dumper twice to get this data.

Do you plan to adjust assessment loader to load data from both files?

Reply:
The flag is temporary. After the development is finished and the new input tested I plan to remove the flag and extract everything that is hidden behind the flag now.

Long-term we can adjust the loader and drop the old `TablesJsonTask`, but this is lower priority at the moment.